roles/wazuh/ansible-wazuh-manager/defaults/main.yml

```diff
@@ -158,11 +158,21 @@ wazuh_manager_monitor_aws:
   interval: '10m'
   run_on_start: 'yes'
   skip_on_error: 'yes'
+  disable_service_block: 'yes' # not present in conf file, only for template condition.
```

Comment:
I would rename `disabled` and `disable_service_block` to `s3_disabled` and `s3_service_disabled` in order to unify the nomenclature.